kieker-extension-cassandra/.settings/qa.eclipse.plugin.checkstyle.prefs

```diff
@@ -0,0 +1,4 @@
+configFilePath=../config/cs-conf.xml
```

Comment:
Also applies to this file and the following

Reply:
Done.